src/Components/Web/src/PublicAPI.Unshipped.txt

```diff
@@ -13,3 +13,10 @@ Microsoft.AspNetCore.Components.Web.MouseEventArgs.MovementX.get -> double
 Microsoft.AspNetCore.Components.Web.MouseEventArgs.MovementX.set -> void
 Microsoft.AspNetCore.Components.Web.MouseEventArgs.MovementY.get -> double
 Microsoft.AspNetCore.Components.Web.MouseEventArgs.MovementY.set -> void
+Microsoft.AspNetCore.Components.IFileDownloader
+Microsoft.AspNetCore.Components.IFileDownloader.SaveAs(string! fileName, byte[]! data) -> System.Threading.Tasks.Task!
+Microsoft.AspNetCore.Components.IFileDownloader.SaveAs(string! fileName, System.IO.Stream! data) -> System.Threading.Tasks.Task!
+Microsoft.AspNetCore.Components.FileDownloader
+Microsoft.AspNetCore.Components.FileDownloader.FileDownloader(Microsoft.JSInterop.IJSRuntime! JSRuntime) -> void
+Microsoft.AspNetCore.Components.FileDownloader.SaveAs(string! fileName, byte[]! data) -> System.Threading.Tasks.Task!
+Microsoft.AspNetCore.Components.FileDownloader.SaveAs(string! fileName, System.IO.Stream! data) -> System.Threading.Tasks.Task!
```

Comment:
I think we'd like to keep the actual implementation private/internal to the framework, just exposing the interface should be sufficient.